File: FouFouFood_ServeurNodeJS/foufoufood_serveur/Order.js
class Order {
    constructor(id, userId, restaurantId, items, totalPrice, status, deliveryAddress, deliveryPartner, createdAt, updatedAt) {
        this.id = id;
        this.userId = userId;
        this.restaurantId = restaurantId;
        this.items = items;
        this.totalPrice = totalPrice;
        this.status = status;
        this.deliveryAddress = deliveryAddress;
        this.deliveryPartner = deliveryPartner;
        this.createdAt = createdAt;
        this.updatedAt = updatedAt;
    }
}

module.exports = Order;

File: foufoufood_client_android/app/src/main/java/com/example/foufoufood/model/Order.kt
package com.example.foufoufood.model
import java.time.LocalDate

data class Order(
    val id: String,
    val userId: String,
    val restaurantId: String,
    val items: List<String>,
    val totalPrice: Float,
    var status: String,
    val deliveryAddress: String,
    val deliveryPartner: String,
    val createdAt: LocalDate,
    var updatedAt: LocalDate
)
